### ISSUE TITLE
Shared notes adds extra lines when using Linux Mint

### ISSUE THREAD

**Issue by kentbborjke** [mconf/bigbluebutton#321]
I am not sure if you think this is worth working on. It is only so far show up when using Linux Mint. In shared notes when typing with formatted text it adds extra strange text (Lines) before a line. I am posting a screen dump to show this. It seems as if you use bigger fonts these lines gets bigger and colored green.

Kent
![shared_notes_bug](https://cloud.githubusercontent.com/assets/6828340/15809037/be383688-2b87-11e6-8232-23234704f42b.jpg)

**Comment by pedrobmarin** [comment]
Hi Kent,

Can you provide us more details about it? Which Mconf build are you running with? Which browsers have you tested and what their Flash's version?

**Comment by kentbborjke** [comment]
Hi Pedro

We use mconf-live 0.7.3.

I tested right now with Firefox on VM, Linux Mint latest version, no pr as far as I can see. I guess it uses latest version of flash, do not know how to check that in Linux mint.

I will test with chrome too.

**Comment by kentbborjke** [comment]
Hi again Pedro

I have tested both chrome and Firefox now on Linux mint and could not reproduce the error. 

this bug showed up on an online meeting, yesterday held from Romania.

I will ask him, what the circumstances was during this error.

I will come back on that

Regards
Kent

**Comment by fcecagno** [comment]
Hi @kentbborjke , can you please test again this on Mconf-Live 0.7.5 and let us know if it persists?

**Comment by kentbborjke** [comment]
Hi Felipe

This problem occurred when I tested shared notes with my friend in Romania. It happened when he used the delete key to the right of the enter button to remove characters. The strange figures showed up only when he used delete key on Linux mint.

I just tested between Linux mint and Linux mint also between Linux mint and windows 7 and it never showed up as before. Do not know why

I have to wait until my friend from Romania comes home in two weeks from now.

It seems to work now as far as I can see.

Will let you know if it comes back or anything else shows up

Regards
Kent
